.changeset/rich-sloths-hang.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-auth-backend-module-microsoft-provider': patch
+'@backstage/plugin-catalog-backend-module-msgraph': patch
+---
+
+Update `catalog.providers.microsoftGraphOrg.target` config def to be optional as this has a default value.
+Update `auth.microsoft.signIn.resolvers` config def to include the `userIdMatchingUserEntityAnnotation` resolver.
```

Comment:
Ah actually, could you split this into two changesets? One for each package and only include the line for which it applies